packages/cosmic-swingset/src/shutdown.js

```diff
@@ -0,0 +1,67 @@
+/* global process */
+// @ts-check
+import anylogger from 'anylogger';
+
+const console = anylogger('shutdown');
+
+export const makeFreshShutdown = () => {
+  const shutdownThunks = new Set();
+
+  let shuttingDown = false;
+  /** @type {NodeJS.SignalsListener & NodeJS.BeforeExitListener} */
+  const shutdown = code => {
+    const sig = typeof code === 'string' && code.startsWith('SIG');
+    if (sig) {
+      process.exitCode = 99;
```

Comment:
> I'm curious, why do we set an exit code for signal based shutdown?

For compatibility.

> Also, I remember seeing `98` being used for this before by `ag-chain-cosmos` (at least that's what the loadgen catches and considers as 0). Why the difference?

Because I (accidentally) broke compatibility.  Fixed now.